### PR TITLE
[RFR][Feature][OSF-598]Utilize linkifyjs for URLifying comments

### PR DESCRIPTION
**Pull request by TomBaxter**
## Purpose:
[OSF-598](https://openscience.atlassian.net/browse/OSF-598)
Update comments to automatically recognize URLs 

## Changes:
Update package.json to require linkifyjs

Update website/static/js/comment.js update var CommentModel to utilize linkifyjs for URL recognition.

## Side effects
None

[#OSF-598]

### CODE REVIEW

**Comment by mfraezz** [comment]
LGTM. Dropping this in RTM 🚲

File: website/static/js/comment.js [hunk 1]
```diff
@@ -327,11 +327,12 @@ var CommentModel = function(data, $parent, $root) {
         self.author = self.$root.author;
     }
 
-    self.contentDisplay = ko.observable(linkifyHtml(markdown.full.render(self.content())));
+    var linkify_opts = { target: function (href, type) { return type === 'url' ? '_top' : null; } };
```
Comment (mfraezz) [inline website/static/js/comment.js]: :+1:

File: website/static/js/comment.js [hunk 2]
```diff
@@ -326,11 +327,12 @@ var CommentModel = function(data, $parent, $root) {
         self.author = self.$root.author;
     }
 
-    self.contentDisplay = ko.observable(markdown.full.render(self.content()));
+    var linkify_opts = { target: function (href, type) { return type === 'url' ? '_top' : null; } };
```
Comment (sloria) [inline website/static/js/comment.js]: As per COSDev, use lowerCamelCase for variable names. I'll fix this when I merge.

**Comment by sloria** [comment]
This is good to merge. Will merge for next improvements release.

**Pull request merged**